### PR TITLE
HTML Footer

### PR DESCRIPTION
**Pull request by bobwise**
## What does this PR do?
- Change `<a>` to `<button>`
- Tests are all in toggle

### Associated Issue
Fixes https://github.com/sparkdesignsystem/spark-design-system/issues/2949

### Code
 - [x] Build Component in HTML
 - [X] Unit Testing in HTML with `npm run test` in `html/` (100% coverage, 100% passing)

### Accessibility
- [X] New changes abide by [accessibility requirements](https://sparkdesignsystem.com/docs/accessibility)

### Browser Testing (current version and 1 prior)
  - [X] Google Chrome
  - [ ] Google Chrome (Mobile)
  - [ ] Mozilla Firefox
  - [ ] Mozilla Firefox (Mobile)
  - [ ] Microsoft Internet Explorer 11 (only this specific version of IE)
  - [ ] Microsoft Edge
  - [ ] Apple Safari
  - [ ] Apple Safari (Mobile)

### CODE REVIEW

File: html/components/footer.stories.js [hunk 1]
```diff
@@ -264,12 +264,12 @@ export const defaultStory = () => {
 
             <div class="sprk-o-Stack__item">
               <div data-sprk-toggle="container">
-                <a class="sprk-b-TypeBodyFour sprk-b-Link sprk-b-Link--simple sprk-b-Link--has-icon sprk-u-FontWeight--normal" data-sprk-toggle="trigger" href="#nogo">
+                <button class="sprk-b-TypeBodyFour sprk-b-Link sprk-b-Link--simple sprk-b-Link--has-icon sprk-u-FontWeight--normal sprk-c-Toggle__trigger" data-sprk-toggle="trigger">
```
Comment (afebbraro) [inline html/components/footer.stories.js]: is it supposed to have all these other classes?

Comment (bobwise) [reply html/components/footer.stories.js]: I think it should still have all of these classes. Semantically this is a button, but stylistically it has not changed. Each of these classes is applying styles that should still be applied.

Comment (guern1kn) [reply html/components/footer.stories.js]: The new toggle__trigger class applies all the same styles as these extra classes. So we won't need them anymore.

Comment (bobwise) [reply html/components/footer.stories.js]: Gotcha. I was mistaken about what RV's change is.

**Pull request merged**